Mathlib/FieldTheory/Differential/Basic.lean

```diff
@@ -89,4 +87,87 @@ lemma _root_.algebraMap.coe_logDeriv {F K : Type*} [Field F] [Field K] [Differen
     [Differential K] [Algebra F K] [DifferentialAlgebra F K]
     (a : F) : logDeriv a = logDeriv (a : K) := (logDeriv_algebraMap a).symm
 
+/--
+The differential of a separable value in a field extension.
+-/
+noncomputable def implicitDeriv' (F : Type*) {K : Type*} [CommRing F] [Differential F] [Field K]
```

Comment:
I wonder whether the name should recall that there is some separability assumption here. Or maybe that definition should be made private, or an abbreviation. In general (over rings, etc.), there would be a condition for the value of this derivative.